crypto/bn/bn_print.c

```diff
@@ -138,7 +138,7 @@ int BN_hex2bn(BIGNUM **bn, const char *a)
         a++;
     }
 
-    for (i = 0; i <= (INT_MAX/4) && isxdigit((unsigned char)a[i]); i++)
+    for (i = 0; i <= (INT_MAX/4) && ossl_isxdigit((unsigned char)a[i]); i++)
```

Comment:
Again, the meaning of the code is unchanged.  I'll try to capture these in a ticket to investigate EBCDIC operation.

`ossl_isXXX` is identical to `isXXX` except on non-EBCDIC systems where the `isXXX` version returns true for characters > 127.